Source/common/SNTCommonEnums.h

```diff
@@ -13,6 +13,7 @@
 ///    limitations under the License.
 
 #import <Foundation/Foundation.h>
+#include <objc/objc-api.h>
```

Comment:
What's this for?

Reply:
I think this is a clangd artifact that I didn't clean up.